### PR TITLE
chore: add README and refine render parameters

### PR DESCRIPTION
**Pull request by mrderyk**
## CONTEXT
The following changes set up the repo and widget files for consumption by our users.

## CHANGES
- Add README
- Eliminate version name from file name for consistent naming/easier filename recall
- Update `renderSearch` function to accept object of params, for easier extensibility

### CODE REVIEW

File: setupBuildConfigs.js [hunk 1]
```diff
@@ -13,7 +12,7 @@ const setupBuildConfigs = () => ({
   target: ["es6", "node12.22.0"],
 
   external: [...Object.keys(devDependencies)],
-  outfile: `../lib/${packageName}-${version}.js`,
+  outfile: `../lib/${packageName}.js`,
```
Comment (mrderyk) [inline setupBuildConfigs.js:15]: We'll add a github action to purge the jsDelivr cache whenever a new file is built.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+>
+> - **[React-Search](https://github.com/vectara/react-search)**: Add Vectara semantic search to your React apps with a few lines of code.
+> - **[React-Chatbot](https://github.com/vectara/react-chatbot)**: Add a Vectara chatbot widget or chat logic to your React apps with a few lines of code.
+> - **[Create-UI](https://github.com/vectara/create-ui)**: The fastest way to generate a working React codebase for a range of generative and semantic search UIs.
+> - **[Vectara Answer](https://github.com/vectara/vectara-answer)**: Demo app for Summarized Semantic Search with advanced configuration options.
+> - **[Vectara Ingest](https://github.com/vectara/vectara-ingest)**: Sample templates and crawlers for pulling data from many popular data sources.
+
+## Usage
+
+### React-Search
```
Comment (cjcenizal) [inline README.md:21]: I was kind of confused by this name. Can we change this to JS-Search?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    customerId: "<your Vectara customer ID>",
+    corpusId: "<your Vectara corpus ID>",
+    apiKey: "<your Vectara query API key>",
+    placeholder?: "<optional string placeholder for your search input>",
+    isDeepLinkable?: <optional boolean specifying if searches should be deeplinkable (modifies page URL)>,
+    openResultsInNewTab?: <optional boolean specifying if clicked results should open in a new tab>
+  });
+```
+
+### React-Chatbot
```
Comment (cjcenizal) [inline README.md:43]: JS-Chatbot?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+> - **[React-Chatbot](https://github.com/vectara/react-chatbot)**: Add a Vectara chatbot widget or chat logic to your React apps with a few lines of code.
+> - **[Create-UI](https://github.com/vectara/create-ui)**: The fastest way to generate a working React codebase for a range of generative and semantic search UIs.
+> - **[Vectara Answer](https://github.com/vectara/vectara-answer)**: Demo app for Summarized Semantic Search with advanced configuration options.
+> - **[Vectara Ingest](https://github.com/vectara/vectara-ingest)**: Sample templates and crawlers for pulling data from many popular data sources.
+
+## Usage
+
+### React-Search
+
+Add this script tag to the `<head>` tag of your website:
```
Comment (cjcenizal) [inline README.md:23]: Above this, can we add this brief description?

> JS-Search is an embeddable form of our [React-Search component](https://github.com/vectara/react-search). Try that component's [demo](https://vectara.github.io/react-search/) to see how this widget will appear and behave on your site.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    apiKey: "<your Vectara query API key>",
+    placeholder?: "<optional string placeholder for your search input>",
+    isDeepLinkable?: <optional boolean specifying if searches should be deeplinkable (modifies page URL)>,
+    openResultsInNewTab?: <optional boolean specifying if clicked results should open in a new tab>
+  });
+```
+
+### React-Chatbot
+
+Add this script tag to the `<head>` tag of your website:
```
Comment (cjcenizal) [inline README.md:45]: > JS-Chatbot is an embeddable form of our [React-Chatbot component](https://github.com/vectara/react-chatbot). Try that component's [demo](https://vectara.github.io/react-chatbot/) to see how this widget will appear and behave on your site.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+> - **[Vectara Answer](https://github.com/vectara/vectara-answer)**: Demo app for Summarized Semantic Search with advanced configuration options.
+> - **[Vectara Ingest](https://github.com/vectara/vectara-ingest)**: Sample templates and crawlers for pulling data from many popular data sources.
+
+## Usage
+
+### React-Search
+
+Add this script tag to the `<head>` tag of your website:
+
+```
```
Comment (cjcenizal) [inline README.md:25]: We can add an annotation to gain syntax highlighting.

```html
<script src="https://cdn.jsdelivr.net/gh/vectara/js-widgets/lib/js-search.js">
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    isDeepLinkable?: <optional boolean specifying if searches should be deeplinkable (modifies page URL)>,
+    openResultsInNewTab?: <optional boolean specifying if clicked results should open in a new tab>
+  });
+```
+
+### React-Chatbot
+
+Add this script tag to the `<head>` tag of your website:
+
+```
```
Comment (cjcenizal) [inline README.md:47]:
```html
<script src="https://cdn.jsdelivr.net/gh/vectara/js-widgets/lib/js-chatbot.js">
```

**Review by cjcenizal — APPROVED** [review]
Looks great! Didn't test locally. Had some non-blocking suggestions.

File: README.md (hunk 2 above)
Comment (mrderyk) [reply README.md:21]: Oh man! I got confused by the naming. This is def. a blocker 😞

**Pull request merged**